recipes/r-nlmixr2est/meta.yaml

```diff
@@ -0,0 +1,138 @@
+{% set version = '2.1.8' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-nlmixr2est
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/nlmixr2est_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/nlmixr2est/nlmixr2est_{{ version }}.tar.gz
+  sha256: bd2c2e6c7922bc86837e28859e8c967bbb72abf90d2ffd98b735c9e883aab4d3
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'           # [win]
+    - '*/Rblas.dll'       # [win]
+    - '*/Rlapack.dll'     # [win]
+
+requirements:
+  build:
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+    - {{ compiler('c') }}          # [not win]
+    - {{ compiler('m2w64_c') }}    # [win]
+    - {{ compiler('cxx') }}        # [not win]
+    - {{ compiler('m2w64_cxx') }}  # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}sed               # [win]
+    - {{ posix }}grep              # [win]
+    - {{ posix }}autoconf
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}automake-wrapper  # [win]
+    - {{ posix }}pkg-config
+    - {{ posix }}make
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - libgomp                      # [linux]
+    - llvm-openmp                  # [osx]
+  host:
+    - r-base
+    - r-bh
+    - r-matrix
+    - r-rcpp
+    - r-rcpparmadillo >=0.11.2.3.1
+    - r-backports
+    - r-checkmate
+    - r-cli
+    - r-knitr
+    - r-lbfgsb3c
+    - r-lotri
+    - r-magrittr
+    - r-minqa
+    - r-n1qn1 >=6.0.1_10
+    - r-nlme
+    - r-nlmixr2data
+    - r-rex
+    - r-rxode2 >=2.0.12
+    - r-rxode2parse >=2.0.11
+    - r-rxode2random >=2.0.9
+    - r-symengine
+    - libblas
+    - liblapack
+  run:
+    - r-base
+    - {{ native }}gcc-libs           # [win]
+    - {{ native }}libwinpthread-git  # [win]
+    - r-bh
```

Comment:
`r-bh` is only needed during the build; commenting it out saves quite a bit of space. Same with `r-stanheaders` (not applicable here).